packages/iris-grid/src/AdvancedFilterCreator.tsx

```diff
@@ -18,7 +18,7 @@ import {
   SortDirection,
   FilterItem,
 } from '@deephaven/jsapi-utils';
-import { ContextActionUtils, Tooltip } from '@deephaven/components';
+import { Button, ContextActionUtils } from '@deephaven/components';
```

Comment:
There's something weird going on when showing advanced filters. I don't see the bug, but when it is showing it appears in the wrong place first, then moves, which is incorrect (and a regression):
![advanced-filter-blip](https://user-images.githubusercontent.com/4505624/196715567-aa08d748-a41e-4b7d-b105-999e24b83d24.gif)